openfisca_core/taxscales.py

```diff
@@ -194,7 +194,9 @@ def calc(self, base, factor = 1, round_base_decimals = None):
         base1 = np.tile(base, (len(self.thresholds), 1)).T
         if isinstance(factor, (float, int)):
             factor = np.ones(len(base)) * factor
-        thresholds1 = np.outer(factor, np.array(self.thresholds + [np.inf]))
+        # thresholds1 = np.outer(factor, np.array(self.thresholds + [np.inf]))
+        # changed to below to avoind NaN creation
```

Comment:
Do we really need to keep this history as comments? `git blame` is there for that kind of use  :)